src/utils.ts

```diff
@@ -1,10 +1,10 @@
 import axios, { AxiosRequestConfig } from 'axios';
 import { Transform } from 'stream';
 
-import { AuthorizationRequest, DbTypes, IDbConnection } from './types';
+import { AuthorizationRequest, DbTypes, IDbConnection, ITdsDriver } from './types';
 
 const cachedDbConnections = new Map<string, { db: any, close: any }>();
-export async function getDb(databaseConfig: IDbConnection, dbType: DbTypes) {
+export async function getDb(databaseConfig: IDbConnection, dbType: DbTypes, tdsDriver?: ITdsDriver) {
```

Comment:
I don't see the need for this parameter here; check my comments in `src/types.ts`. 
Of course, implementing `ITdsDriver` in `IDbConnection` might be more challenging. You'd need to figure out how to handle `cachedConnection`, but I think it would have several advantages:

- Passing driver by string would become automatically deprecated.
- The user won't need to remember to pass an additional parameter to `getDb` specifying which driver to use. If the same connection is being used for both bellboy jobs and `getDb`, the user would otherwise need to specify this parameter multiple times.

Reply:
I will submit an update shortly. Thank you!

Reply:
I pushed an update which moves the `driver` parameter into the connection config. Glad to have your review and feedback.